pkg/restic/common.go

```diff
@@ -17,11 +17,15 @@ limitations under the License.
 package restic
 
 import (
+	"context"
 	"fmt"
 	"os"
 	"strings"
 	"time"
 
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
duplicate import for the same package see line below, not sure if I am missing something but looks like a typo